Backend/Remora.Discord.API.Abstractions/API/Voice/Gateway/Commands/ConnectingResuming/IVoiceResume.cs

```diff
@@ -34,7 +34,7 @@ public interface IVoiceResume : IVoiceGatewayCommand
         /// <summary>
         /// Gets the ID of the server to resume the connection to.
         /// </summary>
-        Snowflake ServerID { get; }
+        Snowflake GuildID { get; }
```

Comment:
Same as for the identify payload.

Reply:
It's the same reason as identify payload.

Reply:
See response there.

Reply:
Revert to `ServerID` field in commit 1a6970f.